example/monitor_edid.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Reads out the EDID information of the monitor.
+
+// ignore_for_file: camel_case_extensions
+
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:win32/win32.dart';
+
+class Size {
+  double width;
+  double height;
+  Size(this.width, this.height);
+}
+
+Size getMonitorSizeInMM() {
+  final guidptr = calloc<GUID>();
+
+  guidptr.ref.setGUID(GUID_CLASS_MONITOR);
```

Comment:
You can use the helper function here to set the GUID: 

```suggestion
  final guidptr = GUIDFromString(GUID_CLASS_MONITOR);
```

Reply:
I didn't know thanks